src/utils/swagger.js

```diff
@@ -46,6 +46,10 @@ export default async (specUrl, { spec, internalUrl, disableBigNumbers, disableCa
   spec = spec || await (await fetch(specUrl)).json()
   const jsonImp = disableBigNumbers ? JSON : JsonBig
 
+  if (new URL(specUrl).origin === 'https://compiler.aepps.com') {
```

Comment:
What if one is not using the hosted compiler?

Is the utils/swagger only supposed to work with the hosted one?

Reply:
good point. can we just check if the url starts with `https://`?

Reply:
I believe this is an issue of compiler deployment because if it deployed over HTTPS and have `"schemes":["http"]` in a swagger file, then the swagger file is incorrect 🤷‍♀️ 

Our compiler is also deployed improperly, but I believe it would be fixed soon and we will be able to remove this.